superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -97,7 +97,8 @@ def __init__(
             self.license = None
             self.homepage = None
             self.build_type = 'catkin' if \
-                yoctoRecipe._get_ros_version(distro) == 1 else 'ament_cmake'
+                yoctoRecipe._get_ros_version(distro.name) == 1 \
```

Comment:
Commit body: `get_ros_version` -> `_get_ros_version()`

Reply:
There's another one to change in the commit body of 100feaea0e34ff469fc429345a7f061fcd49b458 .